glass/observations.py

```diff
@@ -328,3 +330,192 @@ def tomo_nz_gausserr(
     binned_nz *= nz
 
     return binned_nz  # type: ignore[no-any-return]
+
+
+class angular_variable_depth_mask:
+    r"""Variable depth mask for tomographic bins.
+
+    This class allows to create a mask with a different variable depth mask in the angular direction for each tomographic bin.
```

Comment:
Word wrapping for consistency

```suggestion
    This class allows to create a mask with a different variable depth mask in the
    angular direction for each tomographic bin.
```